WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -38,13 +44,22 @@ public class SimulationApiModel
         [JsonProperty(PropertyName = "EndTime", NullValueHandling = NullValueHandling.Ignore)]
         public string EndTime { get; set; }
 
+        [JsonProperty(PropertyName = "StopTime", NullValueHandling = NullValueHandling.Ignore)]
+        public string StopTime { get; set; }
```

Comment:
when creating a simulation, will we pass the EndTime or StopTime?  Could you update the documentation in the repo to describe how these new fields are used, e.g. when creating/editing a simulation?